Editor/AGS.Editor/Utils/PaletteUtilities.cs

```diff
@@ -233,20 +232,20 @@ private static void RemapPixels(byte[] pixels, Color[] oldPal, Color[] newPal, b
         }
 
         // 1.5k lookup table for color matching
-        private static uint[] ColDiffTable = new uint[3 * 128];
+        private static uint[] ColDiffTable = new uint[3 * 512];
```

Comment:
There's a comment about "1.5k table", now it does not reflect the table size. Should be either fixed, or removed.